cookiecutter.json

```diff
@@ -9,7 +9,8 @@
     "license": [
         "Apache Software License 2.0",
         "MIT",
-        "BSD"
+        "BSD",
+        "Other, not Open-source"
```

Comment:
NO, we can't......

We only can support open-source licensed extensions to run on our cloud....